### ISSUE TITLE
Request for reimbursement of customs duty of Fedora 24 DVDs

### ISSUE THREAD

**Issue by sztsian** [FZUG/tickets#1]
The Fedora 24 DVDs landed in Red Hat office. However, this time it costs us CNY 215.5 for customs duty.Currently I have problems asking the Red Hat facility to cover it so I paid it from my own pocket. I hope to get it reimbursed. Invoice will arrive later this week.

**Comment by sztsian** [comment]
Mark this as deferred as I am trying to have the office reimburse this for me.

**Comment by sztsian** [comment]
[fedex_invoice_zsun_20160726.pdf](https://github.com/FZUG/tickets/files/405940/fedex_invoice_zsun_20160726.pdf)

Reimbursement pending on Red Hat. I'll update the status when there is feedback.

**Comment by alick** [comment]
ping?

**Comment by sztsian** [comment]
Close as FIXED. Last week I get the reimburse from Red Hat Beijing R&D.
